src/start.ts

```
@@ -65,6 +66,7 @@ export async function runServer(options: RunServerOptions): Promise<void> {
   )
 
   const serverUrl = `http://localhost:${options.port}`
+  const hostForUrl = options.host || 'localhost';
```

ESLint

[error] 'hostForUrl' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)